src/ucp/core/ucp_worker.h

```diff
@@ -28,6 +28,8 @@
  * because it is common for all cases and protocols (TAG, STREAM). */
 #define UCP_WORKER_HEADROOM_PRIV_SIZE 32
 
+/* Max debug string size for @ref ucp_request_query */
+#define UCP_REQUEST_DEBUG_STRING_MAX 200
```

Comment:
it seems there is no check for max length of string

Reply:
For now it's empirically set, we could add calculation based on number of lanes, protocol and request type

Reply:
So the user will get a rough estimation of possible string size. We can calculate in the future this size, based on number lanes, protocols, etc.